<?xml version="1.0"?>
<launch>
  <arg name="just_play" default="false" />
  <arg name="explore" default="false" />
  <arg name="note" default="d6" />
  <arg name="finger" default="ff" />
  <arg name="runs" default="1" />
  <arg name="repeat" default="1" />
  <arg name="continuous" default="false" />
  <arg name="listen" default="false" />
  <arg name="direction" default="0.0" />
  <arg name="string_position" default="-1.0" />

  <node name="run_episode" pkg="tams_pr2_guzheng" type="run_episode.py" output="screen">
    <param name="just_play" value="$(arg just_play)" />
    <param name="explore" value="$(arg explore)" />
    <param name="note" value="$(arg note)" />
    <param name="finger" value="$(arg finger)" />
    <param name="runs" value="$(arg runs)" />
    <param name="repeat" value="$(arg repeat)" />
    <param name="continuous" value="$(arg continuous)" />
    <param name="listen" value="$(arg listen)" />
    <param name="direction" value="$(arg direction)" />
    <param name="string_position" value="$(arg string_position)" />
  </node>
</launch>
